### PR TITLE
Indicators from file script - updated branch

### PR DESCRIPTION
**Pull request by roysagi**
Status
In Progress

Related Issues
fixes: demisto/etc#21319

Description
Integration gets a file entry ID and fetches indicators from the file.

opening a new branch and PR as the last one got corrupted for an unapparent reason.
(past branch: https://github.com/demisto/content/pull/5465)

### CODE REVIEW

**Comment by roysagi** [comment]
@kirbles19 has approved the doc in the last PR (https://github.com/demisto/content/pull/5465) so I'm adding docs approved

File: Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    return indicator_list
+
+
+def txt_file_to_indicator_list(file_path, auto_detect, default_type):
+    with open(file_path, "r") as fp:
+        file_data = fp.read()
+
+    indicator_list = []
+
+    only_indicator_list = file_data.split()
```
Comment (ronykoz) [inline Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py]:
```suggestion
    raw_splitted_data = file_data.split()
```

Comment (roysagi) [reply Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py]: done

**Comment by roysagi** [comment]
> Lets demo this one please.
> It handles unstructured text, right?

Yup works well.

**Pull request merged**